sdcm/tester.py

```diff
@@ -3050,6 +3050,10 @@ def tearDown(self):
         self.stop_resources()
         self.get_test_failures()
 
+        with silence(parent=self, name='closing decoding queue as needed'):
+            if self.test_config.BACKTRACE_DECODING:
+                self.test_config.DECODING_QUEUE.close()
```

Comment:
We probably want to wrap up this line into try-except block.
We should not fail whole tearDown only if this line fails.

Reply:
> We probably want to wrap up this line into try-except block.
> We should not fail whole tearDown only if this line fails.

That's what silence is doing, plus it would generate an event if it would fail